src/openforms/js/components/admin/form_design/registrations/json_dump/JSONDumpOptionsForm.js

```diff
@@ -58,6 +58,7 @@ const JSONDumpOptionsForm = ({name, label, schema, formData, onChange}) => {
         variables: [],
         fixedMetadataVariables: fixedMetadataVariables || [],
         additionalMetadataVariables: [],
+        transformToList: {},
```

Comment:
I'm not sure why this is an object/mapping keyed by the variable/component key and not just a plain list of strings containing the keys to transform?

The mapping creates problems with the camel case conversion, which you don't have if you just have an array: `transformToList: ['component1', 'component5']` and the backend code should be the same in terms of complexity.